portfolio/src/main/webapp/index.html

```diff
@@ -49,22 +49,27 @@
                 <p>Please feel free to drop some questions, comments, and concerns if you have any:</p>
                 <form action="/my-CQCs" method="POST">
                     <input type="text" name="email-input" placeholder="Your Email"/>
-                    <br/><br/>
+                    <br></br>
```

Comment:
For br statements you typically can. just leave it as 

`<br>`

without closing tags or any slashes! It'll still work.